chart/README.md

```diff
@@ -60,6 +60,7 @@ The helm chart is versioned independently from FusionAuth app releases. However,
 | annotations                     | object | `{}`                                                                                                 | Define annotations for fusionauth Deployment.                                                                                    |
 | app.memory                      | string | `"256M"`                                                                                             | Configures the amount of memory to allocate to the Java VM (sets `FUSIONAUTH_APP_MEMORY`).                                       |
 | app.runtimeMode                 | string | `"development"`                                                                                      | Configures runtime mode (sets `FUSIONAUTH_APP_RUNTIME_MODE`). Must be `development` or `production`.                             |
+| app.silentMode                  | string | `"false"`                                                                                            | Configures silent mode (sets `FUSIONAUTH_APP_SILENT_MODE`). Must be `true` or `false`.                                           |
```

Comment:
I believe this should be `boolean`, with no quotes on `false`

Reply:
You're right. I'm somewhat new to helm and wasn't sure how it would print booleans into the templated output. The lazy (but wrong) solution was to fall back to strings. The type in the schema definition is now changed to `boolean`.
I gave it a quick test with `$ helm template fusionauth .` and the output looked right.